go.mod

```diff
@@ -4,26 +4,11 @@ go 1.17
 
 require (
 	github.com/Shopify/sarama v1.29.1
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
 	github.com/alicebob/miniredis/v2 v2.14.3
 	github.com/apache/thrift v0.15.0
 	github.com/avast/retry-go v3.0.0+incompatible
-	github.com/getsentry/sentry-go v0.11.0
-	github.com/go-kit/kit v0.9.0
-	github.com/go-redis/redis/v8 v8.10.0
-	github.com/gofrs/uuid v3.2.0+incompatible
-	github.com/google/go-cmp v0.5.6
-	github.com/joomcode/errorx v1.0.3
-	github.com/joomcode/redispipe v0.9.4
-	github.com/opentracing/opentracing-go v1.2.0
-	github.com/sony/gobreaker v0.4.1
-	go.uber.org/zap v1.15.0
-	gopkg.in/fsnotify.v1 v1.4.7
-	gopkg.in/yaml.v2 v2.3.0
-)
-
-require (
```

Comment:
Here is a reference to how the likely intended this to work:
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.

https://golang.org/doc/go1.17#go-command

We could sort it to list the indirect ones in a separate require block.

Reply:
I generally don't think people should be babysitting this file.  If the go tool spits it out, it's fine with me.